app/client/components/pages/dashboardPage.jsx

```diff
@@ -76,13 +73,6 @@ class DashboardPage extends React.Component {
       page: page
     });
 
-    const parsed = parseQuery(location.search);
-
-    if (parsed.actok) {
-      console.log("actok", parsed.actok);
-      this.props.activateAccount({ body: { activation_token: parsed.actok } });
-    }
-
```

Comment:
This page isn't accessible without being logged in. `nav.jsx` redirects unauthed users to `authPage.jsx` where activation flow is handled.